kvstore.sh

```diff
@@ -331,3 +347,9 @@ kvstore () {
 }
 kvstore "$@"
 exit $?
+
+## for emacs...
+## Local Variables:
+## sh-basic-offset: 2
+## sh-indentation: 2
+## End:
```

Comment:
Yes. I did some research and found that I could add a .dir-locals.el file to the directory and stuff those items in there. I'll amend the commit if that solution is ok with you.